generic/tests/netperf.py

```diff
@@ -482,6 +482,7 @@ def count_interrupt(name):
         sum = 0
         intr = []
         stat = ssh_cmd(server_ctl, "cat /proc/interrupts |grep %s" % name)
+        stat = stat.strip().split("\n")[-1]
```

Comment:
It is what I guessed it.
```
10:49:58 DEBUG| cat /proc/interrupts |grep virtio.-outpu<f-2.6.0]# cat /proc/interrupts |grep virtio.-output ^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H^H
                ^ this is the first line of session's output
10:49:58 DEBUG| 26: 2 0 PCI-MSI-edge virtio1-output.0
```
The problem of your patch is, if there are more than two virtio devices (or maybe one device has more than one queue), the output of `cat /proc/interrupts |grep virtio.-output` will have more than one item, if only pick `[-1]` out, the number of total interrupts will be calculated incorrectly.

This is an issue of the aexpect module, if you do not want to fix it directly, maybe you can try to workaround this by shortening command to `grep virtio.-output /proc/interrupts` (seems the actual value of `$PS1` is too long).